unison-src/transcripts-using-base/tls.output.md

```diff
@@ -29,7 +29,7 @@ test> match (decodeCert.impl (toUtf8 self_signed_cert_pem) with
   
     ⍟ These new definitions are ok to `add`:
     
-    test.ko630itb5m (Unison bug, unknown term)
```

Comment:
I believe this bug was happening because the old slurping code was filtering watches out of the TypecheckedUnisonFile too aggressively.